mining/worker_test.go

```diff
@@ -516,7 +523,9 @@ func TestGenerateWithoutMessages(t *testing.T) {
 	assert.NoError(t, err)
 
 	assert.Len(t, pool.Pending(), 0) // This is the temporary failure.
-	assert.Len(t, blk.Messages, 0)
+
+	assert.Equal(t, types.MessageCollection{}.Cid(), blk.Messages)
```

Comment:
You generate a CID for an empty collection differently in `GenGen`. Is this a different CID? Could you do this there?  It would be nice to have a `types.EmptyMessagesCID` and `types.EmptyReceiptsCID` defined.